### PR TITLE
ENH: Smoother flow

### PR DESCRIPTION
**Pull request by Eric89GXL**
This should make progress for subjects a bit smoother.

@kambysese can you try it with some new subject when you get a chance?

### CODE REVIEW

File: mnefun/_mnefun.py [hunk 1]
```diff
@@ -1005,21 +1013,27 @@ def run_sss_locally(p, subjects, run_indices):
...
+            pos = _headpos(p, r)
+
+            # filter cHPI signals
+            if p.filter_chpi:
+                t0 = time.time()
+                print('        Filtering cHPI signals ... ', end='')
+                raw = filter_chpi(raw)
+                print('%i sec' % (time.time() - t0,))
+
             # apply maxwell filter
```
Comment (kambysese) [inline mnefun/_mnefun.py]: to be clear we still have option for sss without movement comp here, correct?

Comment (Eric89GXL) [inline mnefun/_mnefun.py]: look at changes to `_headpos`

Comment (kambysese) [inline mnefun/_mnefun.py]: Got it

**Comment by kambysese** [comment]
Just got this 

>File "/home/ktavabi/Projects/mnefun/mnefun/_mnefun.py", line 1894, in do_preprocessing_combined
    phase='zero-double', fir_window='hann')
TypeError: filter() got an unexpected keyword argument 'phase'

**Comment by Eric89GXL** [comment]
You need up to date mne python master

**Comment by kambysese** [comment]
I was on `ilabsbrainteam/svd-reg`

**Comment by Eric89GXL** [comment]
We have to wait until the paper is published, then I can incorporate it in mne-python upstream. In the meantime I need to rebase it myself and force push whenever it needs to be updated. Or you can do the same thing locally if you want.

**Comment by kambysese** [comment]
I've never rebased (successfully) can you please give an example of git command(s) you use to merge `upstream/master` into my local `svd-reg` branch.

**Comment by Eric89GXL** [comment]
This is a decent tutorial:

https://www.atlassian.com/git/tutorials/merging-vs-rebasing/the-golden-rule-of-rebasing

But with merges or rebases you might eventually hit the same conflicts. At least for this branch, the conflicts should be minimal because I don't think many changes to `maxwell_filter.py` have occurred in the `upstream/master` branch.

**Comment by kambysese** [comment]
Thanks I did manage to rebase and created a new local `hot` branch that includes changes in `upstream/master` and `svd-reg`. Hit some conflicts in `epochs.py` that apparently have been resolved! Running `smoother` branch now on own data and it looks good. I'd say +1 for merge.

**Comment by Eric89GXL** [comment]
Thanks @kambysese

**Comment by Eric89GXL** [comment]
FYI @kambysese I rebased `svd-reg` and push-forced

**Comment by kambysese** [comment]
FYI @Eric89GXL just encountered an old assertion error re: mismatch between lengths of `info['bads']` and `cov` index during inversion on your rebased `svd-reg` branch. Error went away with current `upstream/master`

**Pull request merged**